Common/src/main/java/com/blamejared/crafttweaker/api/recipe/component/BuiltinRecipeComponents.java

```diff
@@ -139,6 +142,12 @@ public static final class Output {
                 RecipeComponentEqualityCheckers::areStacksEqual
         );
         
+        public static final IRecipeComponent<List<Percentaged<IItemStack>>> CHANCED_OUTPUTS = IRecipeComponent.simple(
+            CraftTweakerConstants.rl("output/chanced_outputs"),
```

Comment:
Kinda weird to specify output twice
```suggestion
            CraftTweakerConstants.rl("output/chanced_items"),
```